en-US/Players_Guide/05_Spellcasting/Spell_Descriptions/Pelhams_Hovering_Platter.md

```diff
@@ -0,0 +1,19 @@
+#### Pelham's Hovering Platter
+<!-- previously "Floating Disk" -->
+_1st-level conjuration (ritual)_
+
+**Casting Time:** 1 action \
+**Range:** 30 feet \
+**Components:** V, S, M (a drop of mercury) \
+**Duration:** 1 hour
+
+This spell creates a circular, horizontal plane of force, 3 feet in diameter and 1 inch thick, that floats 3 feet above the ground in an unoccupied space of your choice that you can locate within range.
```

Comment:
I swapped "that you can see within range" for "that you can locate within range". I hope, that was correct?